### ISSUE TITLE
V2 : issue with bootstrap on Windows

### ISSUE THREAD

**Issue by z3dev** [jscad/OpenJSCAD.org#765]
## Expected Behavior

The project should bootstrap properly, with simlinks between packages. The web development server should start properly.

## Actual Behavior

Simlinks are missing. The web development server fails to start.

## Steps to Reproduce the Problem

  1. npm install
  2. npm run bootstrap
  3. cd packages/web
  4. npm run dev

## Specifications

  - Version: V2
  - Platform: Windows 10
  - Environment: development environment setup

**Comment by z3dev** [comment]
This seems to be a general issue with lerna.

Here's a work around:
- edit package.json
- remove '--hoist nyc --hoist ava'

The bootstrap should work now.

**Comment by hrgdavor** [comment]
huh I get this error after runing clean again and again:

```
Copying Examples... C:\hrg\3dp_dev\OpenJSCAD.master\packages\web\node_modules\@jscad\examples
ERR: Example examples\import\AMFImport\index.js did not have a valid category set.  Valid categories are Creating Shapes,Manipulating Shapes,Colors,Parameters,Other
ERR: Example examples\import\STLImport\index.js did not have a valid category set.  Valid categories are Creating Shapes,Manipulating Shapes,Colors,Parameters,Other
ERR: Example examples\import\SVGImport\index.js did not have a valid category set.  Valid categories are Creating Shapes,Manipulating Shapes,Colors,Parameters,Other
ERR: Example examples\module-design\index.js did not have a valid category set.  Valid categories are Creating Shapes,Manipulating Shapes,Colors,Parameters,Other
```

**Comment by hrgdavor** [comment]
@z3dev here is possible fix #930 it helped me to build and test again.
